### DISCUSSION TITLE
Welcome to FaceRecognition-GUI-APP Discussions!

### DISCUSSION THREAD

**Discussion by joeVenner** [joeVenner/FaceRecognition-GUI-APP#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sravani-patibanda** [comment]
please share the complete code of age, gender and emotion detection

**Reply by liperrino** [reply]
Hi. I am very interested in what you have done. It is very interesting. Please, I would like to know which technologies you have used, and you have proceeded to obtain this wonderful result?

**Comment by liperrino** [comment]
Hi. I am very interested in what you have done. It is very interesting. Please, I would like to know which technologies you have used, and you have proceeded to obtain this wonderful result?

**Comment by joeVenner** [comment]
Hello Perrin,

I'm very glad to hear that from you!

I tried to keep this project very simple as it's meant for beginners, So
I'm using only OpenCV for face detection and recognition, I used other
libraries like PyTorch & TensorFlow for emotion and gender detection but I
saw that people face problems with them so I removed them. What I can tell
is just go to youtube and type Face Recognition with OpenCV and you can
start making your program in the first 10mns.

Good Luck! and have a great day.
